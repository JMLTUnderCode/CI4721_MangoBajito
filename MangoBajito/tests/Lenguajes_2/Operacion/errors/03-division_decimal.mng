# Test 03: Error de operacion de division decimal.
se_prende(){
	culito n0 : manguita = 8.2 / 2;
	culito n1 : manguita = 8.2 / 1.32e+200;
	culito n2 : manguita = 8.2 / 'c';
	culito n3 : manguita = 8.2 / "hola";
	culito n4 : manguita = 8.2 / Sisa;
	culito n5 : manguita = 8.2 / 2 / 1.2e+100;
};