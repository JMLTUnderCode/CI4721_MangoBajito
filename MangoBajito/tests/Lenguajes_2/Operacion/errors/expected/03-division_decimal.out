| AcomÃ³date que te lo estan analizando d:-)
Error: INVALID_OPERATION
  - Sendo peo en la linea 3, columna 32: Estas operando "/" entre 'manguita' y 'mango', que vaina es loca?
  - Sendo peo en la linea 4, columna 40: Estas operando "/" entre 'manguita' y 'manguangua', que vaina es loca?
  - Sendo peo en la linea 5, columna 34: Estas operando "/" entre 'manguita' y 'negro', que vaina es loca?
  - Sendo peo en la linea 6, columna 37: Estas operando "/" entre 'manguita' y 'higuerote', que vaina es loca?
  - Sendo peo en la linea 7, columna 35: Estas operando "/" entre 'manguita' y 'tas_claro', que vaina es loca?
  - Sendo peo en la linea 8, columna 33: Estas operando "/" entre 'manguita' y 'mango', que vaina es loca?
  - Sendo peo en la linea 8, columna 43: Estas operando "/" entre 'manguita' y 'manguangua', que vaina es loca?
[1;31m[5m
               =======================================================
                       ---->        Error Program        <----        
               =======================================================
[0m
| Hay un poquito de cÃ³digo en tu marihuana d:-]
